var/spack/repos/builtin/packages/py-meld/package.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyMeld(PythonPackage, CudaPackage):
```

Comment:
There is already a `meld` on PyPI and it's not the same one. We may want to change the name of this package.

Reply:
Yeah,,, I didn't know naming precedence for `py-*` packages. Do pypi packages always get naming precedence or is it which ever gets packaged first?

Reply:
That's the policy I've been trying to follow. PyPI is the authoritative source for Python downloads (even Conda downloads tarballs from PyPI), so if two packages have the same GitHub name I always go with the PyPI name to avoid collisions. Although the PyPI meld isn't yet in Spack, it could be added someday, which would require renaming your `py-meld` package.